### PR TITLE
Dockerfile - Add CUDA11.8 Docker image for Nvidia arch90 GPUs

### PR DESCRIPTION
**Pull request by abuccts**
Add Docker image for arch90 NVIDIA GPUs:

* add CUDA11.8 Dockerfile
* update archs in Makefile and benchmarks accordingly
* update image build pipeline

### CODE REVIEW

**Review by yukirora — APPROVED** [review]

File: .github/workflows/build-image.yml [hunk 1]
```diff
@@ -24,6 +24,9 @@ jobs:
     strategy:
```
Comment (cp5555) [inline .github/workflows/build-image.yml:24]: Why not add CUDA 12.0 directly?

**Review by cp5555 — APPROVED** [review]

File: superbench/benchmarks/micro_benchmarks/cuda_gemm_flops_performance.py [hunk 2]
```diff
@@ -47,6 +47,7 @@ def __init__(self, name, parameters=''):
         # Skip FP64 for RTX Turing/Ampere and Tesla T4/GA10x due to very limited FP64 TFLOP rate
```
Comment (cp5555) [inline superbench/benchmarks/micro_benchmarks/cuda_gemm_flops_performance.py:47]: Separated PR

File: dockerfile/cuda11.1.1.dockerfile [hunk 3]
```diff
@@ -22,6 +22,7 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends \
     autoconf \
     automake \
+    bc \
```
Comment (cp5555) [inline dockerfile/cuda11.1.1.dockerfile:25]: What change for this?

**Review by cp5555 — APPROVED** [review]

File: .github/workflows/build-image.yml (hunk 1 above)
Comment (abuccts) [reply .github/workflows/build-image.yml:24]: there's no cuda12.0 image yet, using the [latest one](https://docs.nvidia.com/deeplearning/frameworks/pytorch-release-notes/rel-22-12.html#rel-22-12) which is 11.8

File: dockerfile/cuda11.1.1.dockerfile (hunk 3 above)
Comment (abuccts) [reply dockerfile/cuda11.1.1.dockerfile:25]: to compare the cuda version in makefile, >=11.8 and <11.8 have different build logic

File: superbench/benchmarks/micro_benchmarks/cuda_gemm_flops_performance.py (hunk 2 above)
Comment (abuccts) [reply superbench/benchmarks/micro_benchmarks/cuda_gemm_flops_performance.py]: removed

**Pull request merged**